adafruit_tinylora/adafruit_tinylora.py

```diff
@@ -370,3 +417,6 @@ def _write_u8(self, address, val):
             self._BUFFER[1] = val
             # pylint: disable=no-member
             device.write(self._BUFFER, end=2)
+            # pylint: disable=no-member
+            device.write(self._BUFFER, end=2)
+            device.write(self._BUFFER, end=2)
```

Comment:
Were these lines here duplicated accidentally?

Reply:
Yes. Weirdness with my editor and pre-commit I think. Removed.